### PR TITLE
Styling/ux touchups

### PR DESCRIPTION
**Pull request by mgramigna**
# Summary

## New behavior

No new behavior, screenshots showcasing styling updates shown below:

**Homepage**

image

* Simplified all of the content. The homepage read a lot like a README to me, with instructions on running the apps and stuff which aren't super relevant if the app is already running. Users can look at the GitHub and associated docs if they want more info.
* Reduced the size of the app header and used the default primary blue color instead of the custom cyan (we can always change the colors later, I'm just keeping it default for now)
* Included search button on the homepage as the main call to action. I imagine there will be more options here as we get into the authoring functionality.

**Resource List Page**

image

* Used contrasting background color for emphasis of the cards instead of surrounding them in a box. Used a specific pixel value for width of the paper component for the card.
* Removed back button.
* Moved search down to the center below the title and centered the title properly.

**Resource Page**

image

* Cleaned up text a bit
* Removed back button

**Misc**

Used Open Sans font for the body for now. Could change this later, but wanted to move away from the default

## Code changes

* All the above
* Removed a lot of erroneous stylings that didn't really do anything (inline styles instead of props, unnecessary margins, unnecessary CSS attributes)
* Removed back button component entirely, it's basically the same as hitting the back button in the browser, and doesn't really provide navigation to anywhere that you can't get to in 1 or 2 clicks already

# Testing guidance

Test all features

### CODE REVIEW

**Review by elsaperelli — CHANGES_REQUESTED** [review]
Great updates! I am no UI expert, but I think these changes look great and make sense to me! Just a couple inline comments and one overall suggestion...

Definitely easier said than done, but thoughts on either lining the divider in the main page up with the one in the nav bar OR making the dividers in the main pages all the same spot...right now it's all different:

image

image

image

File: app/src/pages/index.tsx [hunk 1]
```diff
@@ -1,12 +1,55 @@
-import { Anchor, Divider, List, Table, Text, Title } from '@mantine/core';
+import { Anchor, Button, Center, Divider, Group, Table, Text, Title } from '@mantine/core';
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import Link from 'next/link';
 
 export default function Home({ capabilityStatement }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const renderCapabilityTable = () => {
```
[the remark this answers is not in the document]
Comment (mgramigna) [reply app/src/pages/index.tsx:6]: I think we can just leave it as is for now. Maybe will revisit if this makes sense to do once we have two different statuses of artifacts in the app in the first place and see how intuitive it is

File: app/src/components/ResourceButtons.tsx [hunk 2]
```diff
@@ -25,41 +25,28 @@ export default function ResourceButtons({ resourceInfo, resourceType }: Resource
...
+                <div key={res.id}>
+                  <ResourceInfoCard resourceInfo={res} />
+                </div>
+              );
+            })}
+          </Stack>
+        ) : (
+          <Text>
+            No <i>{`${resourceType}`}</i> resources available
+          </Text>
```
[the remark this answers is not in the document]
Comment (mgramigna) [reply app/src/components/ResourceButtons.tsx]: https://github.com/projecttacoma/measure-repository/pull/36/commits/365d4fa2c2cc3ec1ca315440ddc1b9e31fa22f91

File: app/src/components/ResourceCounts.tsx [hunk 3]
```diff
@@ -39,39 +39,31 @@ const ResourceCounts = () => {
...
         <div
           style={{
-            padding: '16px',
-            color: '#E00000',
-            border: '1px solid',
-            borderColor: '#DEE2E6',
-            borderRadius: '20px',
-            marginLeft: '16px',
-            marginRight: '16px'
+            padding: '24px 24px 24px 24px'
```
[the remark this answers is not in the document]
Comment (mgramigna) [reply app/src/components/ResourceCounts.tsx]: https://github.com/projecttacoma/measure-repository/pull/36/commits/365d4fa2c2cc3ec1ca315440ddc1b9e31fa22f91

File: app/src/components/ResourceCounts.tsx [hunk 4]
```diff
@@ -39,39 +39,31 @@ const ResourceCounts = () => {
       return (
```
[the remark this answers is not in the document]
Comment (mgramigna) [reply app/src/components/ResourceCounts.tsx:38]: https://github.com/projecttacoma/measure-repository/pull/36/commits/365d4fa2c2cc3ec1ca315440ddc1b9e31fa22f91

File: app/src/pages/[resourceType]/search-result.tsx [hunk 5]
```diff
@@ -15,38 +14,27 @@ export default function ResourceSearchResultsPage({
...
-          </div>
-        </Grid.Col>
-        <Grid.Col offset={2} span={2} style={{ paddingTop: '6px' }}>
-          <h2 style={{ color: 'gray', marginTop: '0px', marginBottom: '8px' }}>Search Results</h2>
-        </Grid.Col>
-      </Grid>
-      <Divider my="md" style={{ marginTop: '14px' }} />
+      <Center>
+        <Text c="gray" fz="xl">
+          Search Results
```
[the remark this answers is not in the document]
Comment (mgramigna) [reply app/src/pages/[resourceType]/search-result.tsx]: https://github.com/projecttacoma/measure-repository/pull/36/commits/365d4fa2c2cc3ec1ca315440ddc1b9e31fa22f91 (added an indication of how many results came back)

**Review by hossenlopp — APPROVED** [review]
Looks much improved! Thanks for throwing in the README update too.

**Review by elsaperelli — APPROVED** [review]
Great updates!

**Pull request merged**